src/EnergyPlus/ZoneDehumidifier.hh

```diff
@@ -63,142 +63,108 @@ struct EnergyPlusData;
 
 namespace ZoneDehumidifier {
 
+    enum class CondensateOutlet
+    { // Water Systems
+        Invalid = -1,
+        Discarded, // Default mode where water is "lost"
+        ToTank,    // Collect coil condensate from air and store in water storage tank
+        Num = 2
```

Comment:
Not necessary to explicitly assign `Num` to 2.

Reply:
Right.  Done.